paddle/capi/CMakeLists.txt

```diff
@@ -38,10 +37,13 @@ set(PADDLE_CAPI_INFER_LIBS
     paddle_function
     paddle_gserver
     paddle_proto)
-
 cc_library(paddle_capi_whole DEPS paddle_capi ${PADDLE_CAPI_INFER_LIBS})
 
-# No shared library for iOS
+# Link the static library for inference
+cc_library(paddle_nn_engine DEPS paddle_capi paddle_utils paddle_parameter paddle_math paddle_cuda paddle_proto)
+cc_library(paddle_layers DEPS paddle_function paddle_gserver)
```

Comment:
About the libraries' name, how about to name them `paddle_capi_engine` and `paddle_capi_layers`?

Reply:
OK

Reply:
Done.